ceph_bootstrap/exceptions.py

```diff
@@ -6,3 +6,9 @@ class CephNodeHasRolesException(CephBootstrapException):
     def __init__(self, minion_id, roles):
         super(CephNodeHasRolesException, self).__init__(
             "Cannot remove host '{}' because it has roles defined: {}".format(minion_id, roles))
+
+
+class CephNodeHasNoPublicIPv4Exception(CephBootstrapException):
+    def __init__(self, minion_id):
+        super(CephNodeHasNoPublicIPv4Exception, self).__init__(
+            "No public IPv4 address configured on host '{}'".format(minion_id))
```

Comment:
I think the error message should be:
`"Host '{}' FQDN resolves to the loopback interface IP address"`

and the exception class name should be `CephNodeFqdnResolvesToLoopbackException`

Reply:
Fixed